optimum/commands/export/tflite.py

```diff
@@ -0,0 +1,126 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Defines the command line for the export with TensorFlow Lite."""
+
+import subprocess
+from pathlib import Path
+
+from ...exporters import TasksManager
+
+
+def parse_args_tflite(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "-m", "--model", type=str, required=True, help="Model ID on huggingface.co or path on disk to load model from."
+    )
+    required_group.add_argument(
+        "output", type=Path, help="Path indicating the directory where to store generated ONNX model."
+    )
+
+    optional_group = parser.add_argument_group("Optional arguments")
+    optional_group.add_argument(
+        "--task",
+        default="auto",
+        help=(
+            "The task to export the model for. If not specified, the task will be auto-inferred based on the model. Available tasks depend on the model, but are among:"
+            f" {str(list(TasksManager._TASKS_TO_AUTOMODELS.keys()))}. For decoder models, use `xxx-with-past` to export the model using past key values in the decoder."
+        ),
+    )
+    optional_group.add_argument(
+        "--atol",
+        type=float,
+        default=None,
+        help="If specified, the absolute difference tolerance when validating the model. Otherwise, the default atol for the model will be used.",
+    )
+    optional_group.add_argument(
+        "--pad_token_id",
+        type=int,
+        default=None,
+        help=(
+            "This is needed by some models, for some tasks. If not provided, will attempt to use the tokenizer to guess"
+            " it."
+        ),
+    )
+    optional_group.add_argument("--cache_dir", type=str, default=None, help="Path indicating where to store cache.")
+    optional_group.add_argument(
+        "--trust-remote-code",
+        action="store_true",
+        help="Allow to use custom code for the modeling hosted in the model repository. This option should only be set for repositories you trust and in which you have read the code, as it will execute on your local machine arbitrary code present in the model repository.",
+    )
+
+    input_group = parser.add_argument_group("Input shapes")
+    doc_input = "that the TensorFlow Lite will be able to take as input."
+    input_group.add_argument(
+        "--batch_size",
+        type=int,
+        default=1,
+        help=f"Text tasks only. Batch size {doc_input}",
+    )
+    input_group.add_argument(
+        "--sequence_length",
+        type=int,
+        default=None,
+        help=f"Text tasks only. Sequence length {doc_input}",
+    )
+    input_group.add_argument(
+        "--num_choices",
+        type=int,
+        default=None,
+        help=f"Text tasks only. Num choices {doc_input}",
```

Comment:
Maybe something more detailed? 

What about `Text classification tasks only. How many labels to return` or something like this? wdyt?

Reply:
Fixed!